torch/_inductor/codecache.py

```diff
@@ -574,7 +574,7 @@ def get_hash(cls, obj: Any) -> str:
         return sha256_hash(serialized_data)
 
     @classmethod
-    def debug_str(cls, inp: Any) -> str:
+    def debug_lines(cls, inp: Any) -> List[str]:
```

Comment:
Minor, unrelated point: when I ran strobelight on this, this one function is 0.7% of compilation time for FXGraphCache, and another 0.7% for AOTAutogradCache. I was going to fix this by only calling the function when log level was set to debug, but I realize now that since we want these logs for tlparse this is just always going to be a waste of time. 
https://fburl.com/scuba/pyperf_experimental/on_demand/x7pzb7z7

We might want to consider storing debug_lines in the cache entry so that on a cache hit we don't need to recalculate it? That said, maybe 1.5% warm start time is pretty small and won't matter

Reply:
Yes, please plan for things that go into structured trace to be hit all the time. To be honest, we haven't really interrogated how much compile time slow down we are willing to eat with TORCH_TRACE, a random number I'll pull out of my ass is 10%. We don't instrument the costs here but it would be good to do so so we can evaluate how we're doing.

Reply:
> 0.7% of compilation time

I assume the bigger the model is less percentage of compile time this number will be.